packages/components/src/Popover/Popover.mdx

```diff
@@ -0,0 +1,195 @@
+-->
+
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Popover component, consider the
+following:
+
+### Usage
+
+- A popover should be used to introduce a novel UI element or highlight a change
+  to an existing interface.
+- A popover should contain information that is helpful, but not necessary for
+  the user to successfully interact with the page.
+- Once a popover has been shown, it should not show up again once the user has
+  interacted with it.
+- **You must use a `useRef` hook to create a reference point for the popover to
+  latch to.**
+
+### Content
+
+- Popover text content should not exceed 3 lines and can consist of an optional
+  "dismiss" CTA
+
+### Reference Docs
+
+- Documentation on the popper plugin that was used to build the popover
+  component https://popper.js.org/react-popper/v2/
+
+## Props
+
+<Props of={Popover} />
+
+<!--
+  It is not necessary to provide an example of each prop. Use usage examples to
+  highlight key features of a component or particular considerations.
+
+  See `Button.mdx` for a good example of this.
+-->
+
+## preferredPlacement options
+
+You have 5 preferredPlacement options you can use
+
+- `right`
+- `top`
+- `bottom`
+- `auto`
+- `left`
+
+<Playground>
+  {() => {
+    const topRef = useRef();
+    const leftRef = useRef();
+    const rightRef = useRef();
+    const bottomRef = useRef();
+    const [showPopover, setShowPopover] = useState(true);
+    const styles = {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "flex-start",
+      width: "200px",
+    };
+    return (
+      <div style={styles}>
+        <span ref={topRef}>
+          <Button label="Top Reference" />
+        </span>
+        <Popover
+          attachTo={topRef}
+          open={showPopover}
+          dismissible={true}
+          preferredPlacement="top"
+        >
+          <Content>Top Popover</Content>
+        </Popover>
+        <span ref={leftRef}>
+          <Button label="Left Reference" />
+        </span>
+        <Popover
+          attachTo={leftRef}
+          open={showPopover}
+          dismissible={true}
+          preferredPlacement="left"
+        >
+          <Content>Left Popover</Content>
+        </Popover>
+        <span ref={rightRef}>
+          <Button label="Right Reference" />
+        </span>
+        <Popover
+          attachTo={rightRef}
+          open={showPopover}
+          dismissible={true}
+          preferredPlacement="right"
+        >
+          <Content>Right Popover</Content>
+        </Popover>
+        <span ref={bottomRef}>
+          <Button label="Bottom Reference" />
+        </span>
+        <Popover
+          attachTo={bottomRef}
+          open={showPopover}
+          dismissible={true}
+          preferredPlacement="bottom"
+        >
+          <Content>Bottom Popover</Content>
+        </Popover>
+      </div>
+    );
+  }}
+</Playground>
+
+## Dismissible is false
```

Comment:
Dismissible should never be false:

> should contain information that is helpful, but not necessary for the user to successfully interact with the page.

The user should always have a means to remove a Popover, as it actively blocks some portion of the interface and isn't required to use the page.

Reply:
that's a fair point.  I'll remove the optional `dismissible` prop